app/models/description.rb

```diff
@@ -296,42 +276,42 @@ def self.admins_join_table
     "#{table_name}_admins".to_sym
   end
 
-  # Name of the join table used to keep admin groups.
+  # Wrapper around class method of same name
```

Comment:
DRYer; also simplifies test because it's unnecessary to explicitly test the class method if the instance method is covered.